.github/CODEOWNERS

```diff
@@ -1 +1 @@
-*   @seanneumann @AMoo-Miki @rednaksi91 @KrooshalUX @BSFishy @bandinib-amzn
+*   @seanneumann @AMoo-Miki @rednaksi91 @KrooshalUX @BSFishy @bandinib-amzn @ashwin-pc @joshuarrrr
```

Comment:
@BSFishy Can you please check these errors? Can we fix it?

Reply:
Need a repo admin to give them permissions. Will be fixed when that happens

Reply:
cc @seanneumann

Reply:
https://github.com/orgs/opensearch-project/teams/oui/members They're in.